pkg/kubectl/resize.go

```diff
@@ -70,6 +83,19 @@ type ReplicationControllerResizer struct {
 	client.Interface
 }
 
+func CastResizeAsCondition(r Resizer, precondition *ResizePrecondition, namespace, name string, count uint) wait.ConditionFunc {
+	return func() (bool, error) {
+		switch _, err := r.Resize(namespace, name, precondition, count); err.(type) {
+		case nil:
+			return true, nil
+		case ControllerUpdateError:
```

Comment:
I'm trying to avoid wasting 10s when the GET fails because the apiserver is down (though we'd still waste that time if the update fails for the same reason).